engine/runtime-integration-tests/src/test/scala/org/enso/interpreter/test/instrument/RuntimeAsyncCommandsTest.scala

```diff
@@ -404,15 +434,7 @@ class RuntimeAsyncCommandsTest
     )
     // It's possible that ExecutionComplete is from RecomputeContext not from EditFileNotification.
     // If that's the case, then there might be a race in the output produced by the program.
-    var reallyFinished = false
-    iteration = 0
-    while (!reallyFinished && iteration < 50) {
-      val out = context.consumeOut
-      Thread.sleep(100)
-      reallyFinished = out.contains("True")
-      iteration += 1
-    }
-
+    val reallyFinished = context.out.awaitOnText(exact = false, "True")
```

Comment:
```suggestion
    context.out.awaitOnText(exact = false, "True") shouldBe true
```
is relatively short as well. I still like `assertAwaitOnText` a bit more. Probably because it feels more JUnit4 like...